CHANGELOG.md

```diff
@@ -26,14 +30,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - The copyright for all sources is now attributed to Open Enclave SDK contributors.
 - Update Intel DCAP library dependencies to 1.4.1.
 - Update Intel PSW dependencies to 2.6.100.2 on Windows.
-- The stack guard for OE SDK on SGX is turned on. It is also turned on for enclaves.
+- Enable /W2 to treat W2 warnings as errors for Windows builds. Additionally, fix errors which arise from enabling this flag.
```

Comment:
Technically it's the combination of `/W2` and `/WX` to treat level two warnings as errors. I would reword this so that it won't be misread as requiring any action on the app dev's part:
```
Enable `/W2 /WX` on Windows builds by default to treat W2 warnings as errors.
```